### PR TITLE
DOCS-4975 Docs fixes

### PR DESCRIPTION
**Pull request by anxumalo**
This change combines several fixes from the master branch:

- Fix "Digital Workspace" typo in PNG files

- Correct port number in Docker Compose steps

- Add URL for getting quay.io credentials

### CODE REVIEW

File: docs/helm-deployment-aws_kops.md [hunk 1]
```diff
@@ -339,7 +339,7 @@ export EFS_SERVER=<EFS_ID>.efs.<AWS-REGION>.amazonaws.com
 ```
 ### Creating a Docker registry pull secret
 
-Since we need to use protected Docker images from Quay.io, you need access to a secret with credentials to be able to pull those images.
+Since we need to use protected Docker images from Quay.io, you need access to a secret with credentials to be able to pull those images. Alfresco customers can request their credentials by logging a ticket at https://support.alfresco.com.
```
Comment (killerboot) [inline docs/helm-deployment-aws_kops.md]: I'm not entirely sure about this addition. AFAIK if it is a paying customer, the credentials should be generated automatically for their account in salesforce.

**Comment by montgolfiere** [comment]
FYI: 8082 still appears in list of ports within docker-compose readme ... maybe we can re-word sentence to be more generic (for host ports that need to be available).

File: docs/helm-deployment-aws_kops.md (hunk 1 above)
Comment (anxumalo) [reply docs/helm-deployment-aws_kops.md]: That's correct, but customers still need to request their credentials in this way. I copied the change from the master branch after a PR from Support.

**Comment by anxumalo** [comment]
@montgolfiere Thanks for review & reminder - I've changed the wording based on what's already committed to master.

**Pull request merged**